text/0000-futures.md

```diff
@@ -105,13 +105,21 @@ In addition to `async fn` definitions, futures can be built using adapters, much
 like with `Iterator`s. Initially these adapters will be provided entirely "out
 of tree", but eventually they will make their way into the standard library.
 
-Ultimately asynchronous computations are executed by *tasks*, which are
-lightweight threads. In particular, an *executor* is able to "spawn" a
-`()`-producing `Future` as an independent task; these tasks are then
-cooperatively scheduled onto one or more operating system threads. The
-`Executor` trait defines this interface, and the `task` module provides a host
-of related definitions needed when manually implementing `Future`s or
-executors.
+Ultimately asynchronous computations are executed in the form of *tasks*,
+which are comparable to lightweight threads. A task is a `()`-producing `Future`,
+which is owned by an *executor*, and polled to completion while the being pinned.
+
+*executor*s provide the ability to "spawn" such `Future`s.
```

Comment:
This was an issue with the previous text here, but anyway:

Not all executors will provide an ability to "spawn", the common minimal functionality of an executor is just to "run" futures.

Reply:
I thought spawning one future (like `block_on` does) is still spawning. But I can see how it is misleading. Updated